grammar.js

```diff
@@ -365,21 +367,53 @@ module.exports = grammar({
 
     _let_operator: ($) => choice('=', '+=', '-=', '*=', '/=', '%=', '.='),
 
-    let_statement: ($) =>
+    _global_scope: ($) => 'g:',
```

Comment:
This looks like it is unused.

Reply:
Don't know why I separated this in the first place this is just a scope